packages/account/src/Sections/Profile/PhoneVerification/resend-code-timer.tsx

```diff
@@ -47,6 +49,7 @@ const ResendCodeTimer = ({
         if (resend_code_text !== 'Resend code') {
             setShouldShowDidntGetTheCodeModal(true);
         } else {
+            requestEmailVerificationOTP();
```

Comment:
shouldnt we check if timer is 0 before calling this API

Reply:
as of now we disable the button for 60seconds from FE side. There's no way user can click on resend code before the 60s timers goes to 0. I think it's fine for now to check does the timer goes down to 0. When the BE timestamp is ready then we can implement that here 🙏